README.md

```diff
@@ -24,7 +24,9 @@ composer require rapidez/statamic
 ## Install command
 
 The install command will help you set up all the necessary settings.
-It will mainly setup the [Eloquent driver](https://github.com/statamic/eloquent-driver) and publish the necessary vendor from the rapidez/statamic repo. 
+It will mainly setup the [Eloquent driver](https://github.com/statamic/eloquent-driver) and publish the necessary vendor from the rapidez/statamic repo.
+
+Before running the install script, make sure you have an existing User model as Statamic requires this. See: [Statamic docs](https://statamic.dev/tips/storing-users-in-a-database)
```

Comment:
This doesn't explain how you should get you user model back: https://github.com/laravel/laravel/blob/11.x/app/Models/User.php, as we don't provide it from `rapidez/rapidez`: https://github.com/rapidez/rapidez/blob/040ada124619473f7d713aab537d82cea51f0026/.github/workflows/copy-laravel-source.yml#L45. You'll also need the migrations, etc.

Reply:
Changed the readme with a clearer explanation for how and what with the user model!